atom/common/native_mate_converters/net_converter.cc

```diff
@@ -45,8 +45,11 @@ v8::Local<v8::Value> Converter<scoped_refptr<net::X509Certificate>>::ToV8(
   std::string encoded_data;
   net::X509Certificate::GetPEMEncoded(
       val->os_cert_handle(), &encoded_data);
+
   dict.Set("data", encoded_data);
+  dict.Set("issuer", mate::ConvertToV8(isolate, val->issuer()));
```

Comment:
can be just `val->issuer()` , no need to explicitly call `mate::ConvertToV8`. Same for other places in this PR.